sources/web/datalab/static/util.js

```diff
@@ -46,9 +46,30 @@ define(() => {
     request.send();
   }
 
+  // Generate an absolute link to a path on this same Datalab instance.
+  function datalabLink(fullPath) {
+    var basePath = document.body.getAttribute('data-base-url');
```

Comment:
`data-base-url` is the already-defined attribute used by the Jupyter JS code, so we want to reuse that.